.github/workflows/chaos.yml

```diff
@@ -11,8 +11,8 @@ on:
         type: string
 
 jobs:
-  build_custom_cl_image:
-    name: Build custom CL image
+  E2E_Chaos_Build_Custom_Chainlink_Image:
```

Comment:
Do these actually show anywhere now that we have explicit names for all?

I'd prefer we leave them lowercase, to follow the rest of `yml` workflow style, but we should make sure to add an explicit name for each.